### PR TITLE
fix renamed fn image names

### PR DESCRIPTION
**Pull request by mengqiy**

### CODE REVIEW

File: package-examples/wordpress/Kptfile
Comment (frankfarzan) [inline package-examples/wordpress/Kptfile:11]: can you test the examples work as before?

Comment (frankfarzan) [reply package-examples/wordpress/Kptfile:11]: Also, is there any reference to `unstable` remaining in the kpt repo?

**Review by frankfarzan — APPROVED** [review]

File: pkg/api/kptfile/v1/validation_test.go
Comment (mengqiy) [inline pkg/api/kptfile/v1/validation_test.go:169]: Changes in this file doesn't affect the correctness. But I still want to change it to avoid using the old gcr name.

**Comment by mengqiy** [comment]
This PR need @droot's approval as CODEOWNER

**Review by droot — APPROVED** [review]

**Pull request merged**
